### DISCUSSION TITLE
General Settings in Style Manager

### DISCUSSION THREAD

**Discussion by Idaryss** [artf/grapesjs-mjml#381] [Q&A]
Hi, I would like to implement in my grapesjs mjml project a general sector in the style manager where the user can update for exemple the background color of a button and this applies generally to the newsletter (previous and new button).  But the user can still update the background color individually under the normal sector. 

Can you please provide a sandbox where this is implemented that I can check how ? 

Thanks a lot

**Comment by Idaryss** [comment]
@artf do you have any idea in this ? it will really help

**Comment by omerfcinoglu** [comment]
Hello can you get solution? I want to add a special sector but i cant
